conans/test/unittests/model/options_test.py

```diff
@@ -11,7 +11,7 @@ class TestOptions:
 
     @pytest.fixture(autouse=True)
     def _setup(self):
-        options = {"static": [True, False], "optimized": [2, 3, 4], "path": "ANY"}
+        options = {"static": [True, False], "optimized": [2, 3, 4], "path": ["ANY"]}
```

Comment:
This is changing. I am now forcing to declare "ANY" as an element in a list. It simplifies things and is a bit more consistent too.

Reply:
Could you add a test with ANY without the [] to see if the error is good enough and "pin" the behavior?